src/components/LandingPage/LandingPage.module.scss

```diff
@@ -1,16 +1,46 @@
-//color variables
-$primary: #e3afb3;
+@import '../../variables.modules.scss';
 
 //mobile
 .LandingPage {
-    min-height: 100vh;
-    width: 100%;
+    padding: 50px 25px 50px 25px;
```

Comment:
Padding-top (tillsammans med att vi inte fått med body{ margin: 0; } som ligger i en annan PR)
är anledningen till att pilen trycks ner ur vy, de andra ser inte ut att påverka något

Reply:
Jaa tack så är det ju! Om jag lägger till `box-sizing: border-box;` så räknas padding in i höjden och så blir det rätt!

Reply:
Ett tips är att sätta `* { box-sizing: border-box; }` så gäller det för alla element. Brukar underlätta jättemycket :)